packages/service/src/domain/commands/processors/playback-current-command.ts

```diff
@@ -16,32 +16,36 @@ class PlaybackCurrentCommand extends CommandProcessor {
   }
 
   async execute(command: Command): Promise<CommandProcessingResponse> {
-    const includeQueue = command.rawArgs === 'queue';
+    const includeQueue = command.rawArgs?.includes('queue') ?? false;
+    const includePreview = command.rawArgs?.includes('embed') ?? false;
```

Comment:
I've changed default not to include preview because it was annoying (takes a lot of space in chat), but I think it is a nice feature when we explicite want to show preview instead of having to open link in the browser.

Reply:
Okay, let's leave it as it is